lumigator/backend/backend/services/jobs.py

```diff
@@ -381,8 +398,13 @@ async def handle_annotation_job(self, job_id: UUID, request: JobCreate, max_wait
         dataset_filename = self._dataset_service.get_dataset(dataset_id=request.dataset).filename
         dataset_filename = Path(dataset_filename).stem
         dataset_filename = f"{dataset_filename}-annotated.csv"
+        job_status = ""
+
+        try:
+            job_status = await self.wait_for_job_complete(job_id, max_wait_time_sec)
+        except TimeoutError as e:
```

Comment:
If there's a timeout error, I guess we cannot have a value of succeeded after that????

Reply:
👍🏼 I wasn't sure if we wanted to let the flow carry on to the next conditional check so we also got the same log written saying the results weren't going to be stored in the DB.

Reply:
Let's give this place a check, after this PR. The except should probably cover a few following lines.